src/io-methods/azure.js

```diff
@@ -0,0 +1,37 @@
+const azure = require("azure-storage");
+let blobService;
```

Comment:
This variable doesn't seem to be used by anything. Remove?

Reply:
The tests broke when I immediately had the blobService created because this file is statically imported and there is no env variable for azure setup in the tests. I meant to use `getBlobService` to generate the blobService once, I accidentally didn't check if it was already defined instead of recreating it each time.